packages/playground/src/utils/filter_nodes.ts

```diff
@@ -147,3 +157,81 @@ export const inputsInitializer: FilterInputs = {
     type: "text",
   },
 };
+
+export const DedicatedNodeInitializer: DedicatedNodeFilters = {
+  total_cru: {
+    label: "Total CPU (Cores)",
+    placeholder: "Filter by total Cores.",
+    type: "text",
+    rules: [
+      [isDecimal("This Field accepts only a valid number."), min("This Field must be a number larger than 0.", 1)],
+    ],
+  },
+  total_mru: {
+    label: "Total RAM (GB)",
+    placeholder: "Filter by total Memory.",
+    type: "text",
+    rules: [
+      [
+        isDecimal("This Field accepts only a valid number."),
+        min("This Field must be a number larger than 0.", 1),
+        validateResourceMaxNumber("This value is out of range."),
+      ],
+    ],
+  },
+  total_sru: {
+    label: "Total SSD (GB)",
+    placeholder: "Filter by total SSD.",
+    type: "text",
+    rules: [
+      [
+        isDecimal("This Field accepts only a valid number."),
```

Comment:
isDecimal? why it should be Decimal? You can use `isNumeric` to validate if the resource type is number